planning/behavior_velocity_planner/src/scene_module/intersection/scene_intersection.cpp

```diff
@@ -675,13 +675,18 @@ lanelet::ConstLanelets IntersectionModule::getEgoLaneWithNextLane(
   const autoware_auto_planning_msgs::msg::PathWithLaneId & path) const
 {
   const auto & assigned_lanelet = lanelet_map_ptr->laneletLayer.get(lane_id_);
-  const auto last_itr = std::find_if(
-    path.points.crbegin(), path.points.crend(),
-    [this](const auto & p) { return p.lane_ids.front() == lane_id_; });
+  const auto last_itr =
+    std::find_if(path.points.crbegin(), path.points.crend(), [this](const auto & p) {
+      for (const auto lane_id : p.lane_ids) {
+        if (lane_id == lane_id_) {
+          return true;
+        }
+      }
+      return false;
```

Comment:
return `std::find(p.lane_ids.begin(), p.lane_ids.end(), lane_id_) != p.lane_ids.end()` ?

Reply:
You're right. It's much better. Thanks.
https://github.com/autowarefoundation/autoware.universe/pull/1630/commits/9c99e6475a5ba297c1b16d3c18393a5b16315d80